components/x-topic-search/src/SuggestionList.jsx

```diff
@@ -4,26 +4,26 @@ import styles from './TopicSearch.scss';
 import classNames from 'classnames';
 
 export default ({ suggestions, searchTerm, csrfToken, followedTopicIds = [] }) => (
-	<ul className={ classNames(styles["suggestions"]) } aria-live="polite">
+	<ul className={classNames(styles["suggestions"])} aria-live="polite">
 
-		{ suggestions.map((suggestion, index) => (
-			<li className={ classNames(styles["suggestion"]) }
-				key={ index }
+		{ suggestions.map(suggestion => (
```

Comment:
Maybe remove the space after `{` here too.

Reply:
oh! I missed them! Thank you!